src/components/pages/UserManagement/UserManagement.js

```diff
@@ -1,12 +1,97 @@
-import React from 'react';
-import RenderUserManagement from './RenderUserManagement';
+import React, { useEffect, useState } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
 
-function UserManagement({ LoadingComponent }) {
+import { Table } from 'antd';
+
+const columns = [
+  {
+    title: 'Name',
+    dataIndex: 'name',
+    key: 'name',
+    defaultSortOrder: 'descend',
+    sorter: (a, b) => a.name - b.name,
+  },
+  {
+    title: 'Role',
+    dataIndex: 'role',
+    key: 'role',
+    filters: [
+      {
+        text: 'superAdmin',
```

Comment:
Do we want to hide superAdmin from roles displayed and instead only show Admin for anyone with superAdmin or Admin roles?